advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,309 @@
+
+2. Right Click `Enterprise Data Movement Model` and select `Validate` to validate the Model.
+
+3. No error will be displayed if the Model is valid.
+
+![Model Validation](Images/validmodel.png)
+
+4. Right Click `Enterprise Data Movement Model` and select  `Commit` —> `Full`.
+
+![Model Commit](Images/modelcommit.png)
+
+5. Click Ok on `Commit Model` Screen. This will create the `Model` files.
+
+## Execute Copy Replication using Connect CDC MonCon
+
+Connect CDC MonCon is a GUI application, separate from the Connect CDC Director, that you use for monitoring and control functions. We will use Connect CDC MonCon to execute the Copy Replication. Copy Replication will copy all the data from the Source Tables to the Target Tables.
+
+1. Open Connect CDC MonCon.
+
+2. Click `Model` —> `New Model` from Menu.
+
+3. Provide `Hostname/IP` and select the saved `Model` from the drop down list.
+
+![New Model](Images/newmodel.png)
+
+4. In the `Request` Section, select the `Distribution Model` that was created for copying data created in the above steps and right click and select `start`.
+
+![Distribution Model](Images/distributionmodel.png)
+
+5. Click on `Process` to see the progress.
+
+![Process](Images/Process.png)
+
+6. Open pgAdmin once the copy operation is completed successfully and connect the EDB Postgres Advanced Server Target Instance and check the tables and the data is copied to the tables.
+
+![pgAdmin](Images/pgAdmincopydone.png)
+
+![pgAdmin](Images/pgAdmincopydone2.png)
+
+## Create Replication Distribution using Connect CDC Director
+
+Once the Copy replication operation is successful then a Replication option will be executed which will replicate all the changes made on the Source Tables to the Target Tables.
+
+1. Open Connect CDC Director.
+
+2. Create the `Replication Request` from `Distributions` —> `Newly Created Distribution` —> `Requests` —> `New Request` —> `Replication`.
+
+3. Enter the name and press `Ok`.
+
+![Replication Request](Images/replicationrequest.png)
+
+![Replication Request](Images/replicationrequest2.png)
+
+4. Save the changes in the `Model`.
+
+5. Right Click `Enterprise Data Movement Model` and select `Validate` to validate the Model.
+
+6. Right Click `Enterprise Data Movement Model` and select  `Commit` —> `Full`.
+
+7. Click Ok on `Commit Model` Screen. This will create the Model files.
+
+## Execute Replication using Connect CDC MonCon
+
+Replication distribution will replicate all the changes done on the Source Tables to the Target Tables.
+
+1. Open Connect CDC MonCon.
+
+2. As the Model is updated in the Connect CDC Director to add the `Replication Distribution`, click on the `Model Update`, this will add the `Replication Distribution` to the Connect CDC MonCon Interface.
+
+3. The replication request will be displayed once the `Model` is updated successfully.
+
+![Replication Request](Images/replicationrequest3.png)
```

Comment:
We can't really see anything in this image and the customers wouldn't be able to either. Do we need it? if not can it be deleted or is there something here we really want to show.

Reply:
deleted it.